src/middleware/auth.js

```diff
@@ -17,6 +17,20 @@ export async function validateTeacherRole(req, res, next) {
   next()
 }
 
+export async function validateAdminRole(req, res, next) {
+  if (!req.user) {
+    return sendMessageResponse(res, 500, 'Unable to verify user')
```

Comment:
I think it's safe to assume that if the `req.user` doesn't exist, then we are not handling a logged in user. Therefore, I think it'll be better to return a 401 status instead of a 500

Reply:
Changed res 500 to res 401 status

Reply:
I added an else 403 statement